openmdao/recorders/hdf5_recorder.py

```diff
@@ -10,6 +10,8 @@
 from openmdao.recorders.base_recorder import BaseRecorder
 from openmdao.util.record_util import format_iteration_coordinate
 
+om_case_version = 1
```

Comment:
i wonder if this should be in a single location instead of in multiple files...  `recorders/__init__.py` maybe but that seems less visible than might be desirable.  Another possibility might be `recorders/version.py`.  This assumes we are going to keep all the recorders in sync wrt the data format.